source/.editorconfig

```diff
@@ -0,0 +1,133 @@
+# F0.EditorConfig.DotNet
+root = true
+
+[*]
+end_of_line = crlf
+tab_width = 4
+trim_trailing_whitespace = true
+insert_final_newline = true
+
+[*.sln]
+charset = utf-8-bom
+indent_style = tab
+#indent_size = 4
+
+[*.{csproj,vbproj,props,targets}]
+charset = utf-8
+indent_style = space
```

Comment:
Intentional that we have set space here although we use tabs in everything else? Feel like we should be consistent here

Reply:
Intentionally, since _Visual Studio_ doesn't fully respect `.editorconfig` for `*.csproj` files:
`Edit > Advanced > Format Document` on a `*.csproj` file results in
```xml
<Project Sdk="Microsoft.NET.Sdk">
  <PropertyGroup> <!--2 spaces-->
	<TargetFramework>netstandard2.0</TargetFramework> <!--1 tab-->
  </PropertyGroup> <!--2 spaces-->
</Project>
```

`Tabify` via the [Fix Mixed Tabs](https://marketplace.visualstudio.com/items?itemName=VisualStudioPlatformTeam.FixMixedTabs) extension has the same effect.

However, _Rider_ does fully adhere to the defined `indent_style` and "tabifys" the `.csproj` file accordingly:
```xml
<Project Sdk="Microsoft.NET.Sdk">
	<PropertyGroup> <!--1 tab-->
		<TargetFramework>netstandard2.0</TargetFramework><!--2 tabs-->
	</PropertyGroup> <!--1 tab-->
</Project>
```

Reply:
Is this really the case? Have never used editorconfig before so I tried it out right now with the default template for the config and it was applied correctly on the csproj file. 
Also I found [this issue](https://developercommunity.visualstudio.com/content/problem/159834/editorconfig-doesnt-work-for-csproj-files.html) in the vs dev community and it has been fixed since 2018

Reply:
Indeed, I just tried again and it worked perfectly.
Not sure what I did wrong last time. Perhaps not restarting _Visual Studio_ after editing the `.editorconfig` ...
... trying to cover up the layer 8 issue ;-)